app/src/behaviors/behavior_bt.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 Peter Johanson <peter@peterjohanson.com>
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_bluetooth
+
+#include <device.h>
+#include <drivers/behavior.h>
+
+#include <dt-bindings/zmk/bt.h>
+
+#include <bluetooth/conn.h>
+
+#include <logging/log.h>
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+#include <zmk/ble.h>
+
+static int on_keymap_binding_pressed(struct device *dev, u32_t position, u32_t command, u32_t arg)
+{
+    switch (command)
+    {
+    case BT_RST_CMD:
+        return zmk_ble_unpair_all();
+    case BT_IDENT_CLR_CMD:
+        return zmk_ble_identity_clear();
+#if CONFIG_BT_ID_MAX != 1
+    case BT_IDENT_NEXT_CMD:
+        return zmk_ble_identity_next();
+    case BT_IDENT_PREV_CMD:
+        return zmk_ble_identity_prev();
+    case BT_IDENT_SEL_CMD:
+        return zmk_ble_identity_select(arg);
+#endif /* BT_ID_MAX != 1 */
```

Comment:
default case with logging cmd might be useful while debugging

Reply:
Added as part of the refactor. Good suggestion.